conf/messages.en

```diff
@@ -1239,6 +1239,9 @@ messages__visuallyhidden__dynamic_hasCrn = whether {0} has a company number
 messages__visuallyhidden__dynamic_noCrnReason = the reason why {0} does not have a company number
 messages__visuallyhidden__dynamic_crn = the company number for {0}
 
+messages__dynamic_whatIsUTR = What is {0}’s UTR?
```

Comment:
Content does not match prototype: https://pods-dev-prototype.herokuapp.com/pages/establisher/individual/numbers/utr-yes

Reply:
Content updated